index_releases_sort.go

```diff
@@ -0,0 +1,15 @@
+package versionbundle
+
+import (
+	"github.com/coreos/go-semver/semver"
+)
+
+type SortIndexReleasesByVersion []Release
```

Comment:
I think this should be `[]IndexRelease`.

Reply:
According to go stdlib `ReleaseSlice`: https://golang.org/pkg/sort/#IntSlice

But maybe there can be different approaches to sorting, then this generic name doesn't work.

Reply:
Indeed, changed to the proper type.